catroid/src/org/catrobat/catroid/content/XmlHeader.java

```diff
@@ -67,94 +67,96 @@
 	@SuppressWarnings("unused")
 	private String userHandle = "";
 
+	@SuppressWarnings("PMD.UnnecessaryConstructor")
 	public XmlHeader() {
+		super();
 	}
 
-	String getProgramName() {
+	public String getProgramName() {
```

Comment:
Found it ^^ It's the DefaultPackage rule. You should exclude it.
http://pmd.sourceforge.net/pmd-5.1.1/rules/java/controversial.html#DefaultPackage